src/components/auth/profile/DonationTab.tsx

```diff
@@ -8,198 +12,291 @@ import TableBody from '@mui/material/TableBody'
+              <Grid item xs={6} sm={3}>
+                <CheckboxLabel>{t('auth:profile.donations.oneTime')}</CheckboxLabel>
+                <Checkbox
+                  onChange={(e, checked) => {
+                    setOneTime(checked)
+                  }}
+                  checked={oneTime}
+                  name="oneTime"
+                />
+              </Grid>
+              <Grid item xs={6} sm={3}>
+                <CheckboxLabel>{t('auth:profile.donations.monthly')}</CheckboxLabel>
+                <Checkbox
+                  onChange={(e, checked) => {
+                    setMonthly(checked)
+                  }}
+                  checked={monthly}
+                  name="monthly"
+                />
+              </Grid>
+              <LocalizationProvider
+                locale={i18n.language === 'bg' ? bg : enUS}
+                dateAdapter={AdapterDateFns}>
+                <Grid item xs={12} sm={3}>
+                  <DatePicker
+                    label={t('auth:profile.donations.fromDate')}
+                    value={fromDate}
+                    onChange={setFromDate}
+                    renderInput={(params) => <TextField size="small" {...params} />}
+                  />
+                </Grid>
+                <Grid item xs={12} sm={3}>
+                  <DatePicker
+                    label={t('auth:profile.donations.toDate')}
+                    value={toDate}
+                    onChange={setToDate}
+                    renderInput={(params) => <TextField size="small" {...params} />}
+                  />
+                </Grid>
+              </LocalizationProvider>
+            </Grid>
+            {filteredDonations?.length ? (
+              <TableContainer>
+                <Table sx={{ minWidth: 650, backgroundColor: 'white' }} aria-label="simple table">
+                  <TableHead>
+                    <TableRow>
+                      <TableCell>№</TableCell>
+                      <TableCell>{t('auth:profile.donations.date')}</TableCell>
+                      <TableCell>{t('auth:profile.donations.type')}</TableCell>
+                      <TableCell>{t('auth:profile.donations.cause')}</TableCell>
+                      <TableCell>{t('auth:profile.donations.amount')}</TableCell>
+                      <TableCell>{t('auth:profile.donations.certificate')}</TableCell>
+                    </TableRow>
+                  </TableHead>
+                  <TableBody>
+                    {filteredDonations.map((donation, index) => (
+                      <TableRow
+                        key={index}
+                        sx={{ '&:last-child td, &:last-child th': { border: 0 } }}>
+                        <TableCell component="th" scope="row">
+                          {index + 1}
+                        </TableCell>
+                        <TableCell>
+                          {format(parseISO(donation.createdAt), 'd.LL.yyyy', {
```

Comment:
Please use standardized date format not added on the fly

We already have functions to format the date in specific format or the format as const.

If we don't do that we'll end up having different formatting on different pages

Reply:
Done in https://github.com/podkrepi-bg/frontend/pull/719/commits/15d0d06b172ace6de860f1dc6336566ff4dad008